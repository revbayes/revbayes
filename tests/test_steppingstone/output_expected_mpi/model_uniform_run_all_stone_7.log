Iteration	Posterior	Likelihood	Prior	alpha	bl[1]	bl[2]	bl[3]	bl[4]	bl[5]	bl[6]	bl[7]	bl[8]	bl[9]	bl[10]	bl[11]	bl[12]	bl[13]	bl[14]	bl[15]	bl[16]	bl[17]	bl[18]	bl[19]	bl[20]	bl[21]	bl[22]	bl[23]	bl[24]	bl[25]	bl[26]	bl[27]	bl[28]	bl[29]	bl[30]	bl[31]	bl[32]	bl[33]	bl[34]	bl[35]	bl[36]	bl[37]	bl[38]	bl[39]	bl[40]	bl[41]	bl[42]	bl[43]	er[1]	er[2]	er[3]	er[4]	er[5]	er[6]	gamma_rates[1]	gamma_rates[2]	gamma_rates[3]	gamma_rates[4]	pi[1]	pi[2]	pi[3]	pi[4]	pinvar	TL
0	-24847.28	-24834.56	-12.71838	1.400108	0.101014	0.0791071	0.0444955	0.213955	0.120936	0.257455	0.458798	0.218168	0.0847104	0.259241	0.226002	0.267437	0.165058	0.0871249	0.102414	0.165543	0.284785	0.025548	0.11431	0.133921	0.139544	0.209597	0.174701	0.0867394	0.0280554	0.00395858	0.126881	0.0908904	0.0106278	0.0851508	0.165785	0.135796	0.118122	0.000624515	0.105218	0.086444	0.0827397	0.18935	0.00480863	0.026193	0.0501143	0.203384	0.421794	0.0408878	0.308004	0.0722416	0.0561779	0.477454	0.0452346	0.209471	0.571038	1.04397	2.17552	0.273028	0.267216	0.108942	0.350814	0.3140545	5.956542
10	-25921.62	-25911.35	-10.26738	1.965478	0.0731961	0.180657	0.0228128	0.153754	0.138195	0.234845	0.598203	0.122685	0.145296	0.0601749	0.151401	0.135202	0.186043	0.0797138	0.131562	0.092848	0.223586	0.0305263	0.151516	0.114254	0.0404062	0.590771	0.268804	0.0421704	0.103923	0.00417369	0.142603	0.127008	0.013695	0.0286872	0.14229	0.137349	0.0542149	0.00124071	0.109863	0.199406	0.0475398	0.156712	0.00496778	0.0282864	0.0872859	0.0878464	0.250192	0.191947	0.366851	0.195235	0.0242081	0.173455	0.0483036	0.289119	0.651292	1.0691	1.99049	0.171493	0.4195	0.0866501	0.322357	0.172145	5.695906
