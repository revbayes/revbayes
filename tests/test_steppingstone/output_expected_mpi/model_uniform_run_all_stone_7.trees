Iteration	Posterior	Likelihood	Prior	psi
0	-24847.28	-24834.56	-12.71838	((Varecia_variegata_variegata[&index=22]:0.209597,((Tarsius_syrichta[&index=21]:0.139544,((((((((Otolemur_crassicaudatus[&index=16]:0.165543,(Callicebus_donacophilus[&index=2]:0.079107,Saimiri_sciureus[&index=20]:0.133921)[&index=33]:0.118122)[&index=24]:0.086739,(Cebus_albifrons[&index=3]:0.044496,((Nycticebus_coucang[&index=15]:0.102414,Loris_tardigradus[&index=12]:0.267437)[&index=28]:0.090890,((Lepilemur_hubbardorum[&index=11]:0.226002,Galago_senegalensis[&index=8]:0.218168)[&index=39]:0.004809,Perodicticus_potto[&index=18]:0.025548)[&index=41]:0.050114)[&index=29]:0.010628)[&index=26]:0.003959)[&index=43]:0.421794,Macaca_mulatta[&index=13]:0.165058)[&index=37]:0.082740,Aotus_trivirgatus[&index=1]:0.101014)[&index=25]:0.028055,Chlorocebus_aethiops[&index=5]:0.120936)[&index=42]:0.203384,(Hylobates_lar[&index=9]:0.084710,Pan_paniscus[&index=17]:0.284785)[&index=32]:0.135796)[&index=30]:0.085151,Colobus_guereza[&index=6]:0.257455)[&index=38]:0.189350,Propithecus_coquereli[&index=19]:0.114310)[&index=31]:0.165785)[&index=36]:0.086444,Daubentonia_madagascariensis[&index=7]:0.458798)[&index=27]:0.126881)[&index=34]:0.000625,Galeopterus_variegatus[&index=23]:0.174701,(Cheirogaleus_major[&index=4]:0.213955,(Microcebus_murinus[&index=14]:0.087125,Lemur_catta[&index=10]:0.259241)[&index=40]:0.026193)[&index=35]:0.105218)[&index=44];
10	-25921.62	-25911.35	-10.26738	(((Varecia_variegata_variegata[&index=22]:0.590771,(Propithecus_coquereli[&index=19]:0.151516,(((((Chlorocebus_aethiops[&index=5]:0.138195,(Macaca_mulatta[&index=13]:0.186043,(Tarsius_syrichta[&index=21]:0.040406,(Colobus_guereza[&index=6]:0.234845,(Aotus_trivirgatus[&index=1]:0.073196,Hylobates_lar[&index=9]:0.145296)[&index=41]:0.087286)[&index=32]:0.137349)[&index=36]:0.199406)[&index=37]:0.047540)[&index=25]:0.103923,Saimiri_sciureus[&index=20]:0.114254)[&index=33]:0.054215,(Loris_tardigradus[&index=12]:0.135202,Nycticebus_coucang[&index=15]:0.131562)[&index=28]:0.127008)[&index=24]:0.042170,((Cebus_albifrons[&index=3]:0.022813,Callicebus_donacophilus[&index=2]:0.180657)[&index=43]:0.250192,((Otolemur_crassicaudatus[&index=16]:0.092848,Lemur_catta[&index=10]:0.060175)[&index=29]:0.013695,Perodicticus_potto[&index=18]:0.030526)[&index=40]:0.028286)[&index=26]:0.004174)[&index=30]:0.028687,(Daubentonia_madagascariensis[&index=7]:0.598203,(Lepilemur_hubbardorum[&index=11]:0.151401,Galago_senegalensis[&index=8]:0.122685)[&index=39]:0.004968)[&index=31]:0.142290)[&index=38]:0.156712)[&index=35]:0.109863)[&index=34]:0.001241,Microcebus_murinus[&index=14]:0.079714)[&index=27]:0.142603,(Cheirogaleus_major[&index=4]:0.153754,Pan_paniscus[&index=17]:0.223586)[&index=42]:0.087846,Galeopterus_variegatus[&index=23]:0.268804)[&index=44];
