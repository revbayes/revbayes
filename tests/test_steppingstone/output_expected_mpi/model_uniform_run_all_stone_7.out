state	power	likelihood
4	0.00243	-25851.6
5	0.00243	-25754.3
6	0.00243	-25459.3
7	0.00243	-25643
8	0.00243	-26227.6
9	0.00243	-25892.3
10	0.00243	-25911.3
